README.md

```diff
@@ -70,6 +94,28 @@ everything apart from the Apache front-end units. There is an example of using J
 
 ## Development Environment
 
+### Contributing
```

Comment:
This feels like general GitHub "filing a PR" docs - do they have something we can just refer to from here?

Reply:
I have a look if I can find something in the GitHub documentation I can refer